src/rsc/server-storyblok-rich-text.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef } from 'react';
+
+import { convertAttributesInElement } from '../utils';
+import type { StoryblokRichTextProps } from '../types';
+import { useStoryblokServerRichText } from './server-richtext';
+
+const StoryblokRichText = forwardRef<HTMLDivElement, StoryblokRichTextProps>(
+  ({ doc, resolvers }, ref) => {
```

Comment:
I would suggest to return immediately if we don't have the `doc`, something like

```
    if (!doc) {
      return null;
    }
```